boards/posix/doc/bsim_boards_design.rst

```diff
@@ -7,7 +7,7 @@ This page covers the design, architecture and rationale, of the
 :ref:`nrf52_bsim<nrf52_bsim>` board and other similar bsim boards.
 Particular details on the nRF52 simulation board, including how to use it,
 can be found in that :ref:`board documentation<nrf52_bsim>`.
-These boards are postfixed with `_bsim` as they use BabbleSim
+These boards are postfixed with `_bsim` as they use BabbleSim_
```

Comment:
Is the ending `_` intentional?

Reply:
This comment also applies to several places further below

Reply:
Yes, that makes it a reference to the `BabbleSim` link defined a few lines lower.
Meaning that text ends up as a link to https://babblesim.github.io/.

Reply:
Does that also work when you do something like `\`BabbleSim\`_`?